daskml/preprocessing/data.py

```diff
@@ -45,3 +48,73 @@ def inverse_transform(self, X, copy=None):
         if self.with_mean:
             X += self.mean_
         return X
+
+
+class MinMaxScaler(skdata.MinMaxScaler):
+
+    def __init__(self, feature_range=(0, 1), copy=True, columns=None):
+        super().__init__(feature_range, copy)
+        self.columns = columns
+
+        if not copy:
+            raise NotImplementedError()
+
+    def fit(self, X, y=None):
+        self._reset()
+        self._cache = dict()
+        to_persist = OrderedDict()
+        feature_range = self.feature_range
+
+        if feature_range[0] >= feature_range[1]:
+            raise ValueError("Minimum of desired feature "
+                             "range must be smaller than maximum.")
+
+        _X = slice_columns(X, self.columns)
+        data_min = _X.min(0)
+        data_max = _X.max(0)
+        data_range = data_max - data_min
+        scale = ((feature_range[1] - feature_range[0]) /
+                 handle_zeros_in_scale(data_range))
+
+        to_persist["data_min_"] = data_min
+        to_persist["data_max_"] = data_max
+        to_persist["data_range_"] = data_range
+        to_persist["scale_"] = scale
+        to_persist["min_"] = feature_range[0] - data_min * scale
+        to_persist["n_samples_seen_"] = np.nan
+
+        values = persist(*to_persist.values(), cache=self._cache)
```

Comment:
Cache is almost never used any longer with Dask.  I'm curious what the objective here was and if there is a way to replace this with something else.

Reply:
I used it in my implementation of `StandardScaler`. Looking back now, it seems unnecessary. The call to `persist` should be sufficient to ensure that the values don't have to be re-computed, correct?

Reply:
Yes, `persist` will return dask collections that are equivalent to what were put in, except that now all of the parts will be fully completed.

Reply:
Great. I'll remove it from both of them. Ty.